Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3
+
+COPY magicked_admin /magicked_admin
+
+RUN pip install colorama termcolor lxml requests

```

Comment:
This line should do `pip install -r requirements.txt`, certifi is also needed for HTTPS support.